examples/plot_parula_contours.py

```diff
@@ -16,8 +16,8 @@
 import niceplots
 
 
-niceplots.setRCParams()
-niceColors = niceplots.get_niceColors()
+plt.style.use(niceplots.get_style())
+grey = "#5a5758"
```

Comment:
I think we should figure out a way to keep some colours like this in the dictionary returned by `get_colors` without them being in the colour cycle. It could be specific colours like this (e.g `niceColors["Gray"]`) or it could be style dependent (e.g `niceColours["Axis"]`, `niceColours["Background"]` etc)